src/fsharp/CompileOptions.fs

```diff
@@ -655,6 +655,27 @@ let fullPathsFlag  (tcConfigB : TcConfigBuilder)  =
 let cliRootFlag (_tcConfigB : TcConfigBuilder) = 
         CompilerOption("cliroot", tagString, OptionString (fun _  -> ()), Some(DeprecatedCommandLineOptionFull(FSComp.SR.optsClirootDeprecatedMsg(), rangeCmdArgs)),
                            Some(FSComp.SR.optsClirootDescription()))
+
+let parseCultureName (s: string) =
+    try
+        let culture = new Globalization.CultureInfo(s)
+        if (culture.CultureTypes.HasFlag(Globalization.CultureTypes.UserCustomCulture)) then
+            // Do not use user custom cultures.
```

Comment:
Further the error message will be misleading, as the language name will be actually valid, but will be reported as not.